scripts/create-component/create-component.ts

```diff
@@ -141,26 +137,28 @@ module.exports = (plop: NodePlopAPI) => {
           data,
           skipIfExists: true,
           skip: () => {
-            if (!data.hasStorybook) {
-              return 'Skipping storybook scaffolding';
-            }
+            if (!data.hasStorybook) return 'Skipping storybook scaffolding';
           },
           base: 'plop-templates-storybook',
           templateFiles: [`${renderString(templatePaths.storybookComponent)}/**/*`],
         },
         () => {
-          console.log('\nPackage files created! Running yarn to link...\n');
+          console.log(
```

Comment:
Not essential but I'd prefer to stick with the previous version. At least to me it doesn't seem worth introducing this helper function just for a couple newlines.